app/transformations/MemberPaymentsTransformer.scala

```diff
@@ -135,12 +135,29 @@ class MemberPaymentsTransformer @Inject()(
       }
 
       // Omit memberPSRVersion if member has changed
+      // Check for empty member payment sections before comparing members
       val memberDetailsWithCorrectVersion: List[MemberDetails] = memberDetailsWithCorrectState.map {
         case (index, currentMemberDetail) =>
           val optInitialMemberDetail = initialMemberDetails.get(index)
+          val refinedCurrentMember =
```

Comment:
This change (and related changes in other files) seem not related to the ticket, is it intentional to have it in this PR?

Reply:
Nope accidentally added things from another branch here, removed now